stand_up_bot/services.py

```diff
@@ -37,6 +43,48 @@ def __load_messages(self, filename):
         return rows
 
 
+class StandUpTemplateRepository:
```

Comment:
This is the interface for `StandUpTemplate` repositories. This PR introduces the StandUpTemplateEnvRepository which loads templates from an environment variable. 

In the long term, we might implement other repository types (eg. DynamoDB, PostgresSQL, MongoDB, Redis,...).